impeller/entity/contents/contents.cc

```diff
@@ -107,6 +113,12 @@ std::optional<Snapshot> Contents::RenderToSnapshot(
   if (!render_target.ok()) {
     return std::nullopt;
   }
+  if (!renderer.GetContext()
+           ->GetCommandQueue()
+           ->Submit({std::move(command_buffer)})
```

Comment:
Can you put the explicit type here please. ( https://google.github.io/styleguide/cppguide.html#Type_deduction )

Reply:
In the initializer list?

Reply:
Oh, it's for a vector.  I was thinking it was constructing something like a `SubmitInfo` and we were using the initializer syntax to avoid explicitly mentioning that type.  Man, c++.  I feel different about using it for std::vectors.  It would be a big pain to have to explicitly list those types.  I think `/*buffers=*/` would be an improvement since that's typically what we do for rvalues in parameter lists.